lib/routes/release-line.js

```diff
@@ -0,0 +1,19 @@
+const a = require('express-async-handler');
```

Comment:
I think we should name this something with more letters so it is easier to find when scanning, but I also can't think of anything meaningful.

Reply:
`asyncIt`, `makeItAsync`, `asyncify`, `doItAsync`, `asunk`

Reply:
@fritzmonkey @msluther what about `asuck`